docs/cpu-specific-build-macros.md

```diff
@@ -60,6 +60,12 @@ For Cortex-A53, following errata build flags are defined :
      CPU. This needs to be enabled only for revision <= r0p3 of the CPU. From
      r0p4 and onwards, this errata is enabled by default.
 
+*   `ERRATA_A53_855873`: This applies errata 855873 workaround to Cortex-A53
+     CPUs. Though the erratum is present in every revision of the CPU,
+     this workaround is only applied to CPUs from r0p3 onwards, which feature
+     a chicken bit in CPUACTLR_EL1 to enable a hardware workaround.
+     Earlier revisions of the CPU are meant to be patched by the rich OS.
```

Comment:
What I meant to say is that earlier revision of the core suffer from other errata, which need an equivalent software workaround in _every_ code that is running, possibly by patching code. Linux handles this like this: https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=301bcfac42897dbd1b0b3c1be49f24654a1bc49e
Cortex-A53 r0p3 fixed those errata, also introduced a chicken bit to avoid code patching in the future, just in case. As it turns out we can now use this feature to fix the (new) erratum 855873.